src/Models/Environment.php

```diff
@@ -900,11 +900,14 @@ public function wake()
         $domains = array_filter(
             $this->getDomains()->all(),
             function ($domain) {
-                return !empty($domain->get('dns_zone_name'));
+                $domain_type = $domain->get('type');
+                if (empty($domain_type) || "platform" != $domain_type)
+                    return false;
```

Comment:
To me it seems more readable to just `return <logic>` rather than putting the logic in an if. Optional.

Reply:
What do you suggest?  We needed to test for the presence of the value and the value being equal to "platform".